src/app/utils/index.js

```diff
@@ -0,0 +1,2 @@
+export const BASE_URL = `http://localhost:3000/api`;
```

Comment:
what about consuming this from an environment file? defaulting to localhost is OK.